packages/ui/Scrollable/utilDOM.js

```diff
@@ -0,0 +1,56 @@
+import warning from 'warning';
+
+// At the end, wes should not use DOM APIs as well as the APIs in the `utilDOM`. If we use them, we have to try to remove them first if possible.
+
+const utilDOM = () => {
+	// Functions
+
+	function containsDangerously (ref, target) {
```

Comment:
I don't know well. But Is there any reason it should be a function?
I prefer just object to avoid make duplicated function.
const utilDOM = {
  containsDangerously  : (ref, target) => {...},
  ...
}